app/features/Contents/TradingSidebar/AssetAmount/useAssetAmount.ts

```diff
@@ -44,6 +43,16 @@ export default function useAssetAmount() {
     placeholder: "0",
   };
 
+  // calculate quote amount from base amount and mark price (the terms of
+  // base/quote used in the application are the opposite, this needs to be fixed)
```

Comment:
Maybe I would include `TODO:` in the comment for now then, it's easier to look for those types of comments.